ui/objects/semantic-ui/js/objects/search/components/ResultsListItem.jsx

```diff
@@ -80,6 +85,34 @@ export const ResultsListItemComponent = ({ result, appName }) => {
     };
```

Comment:
Way too complicated way of hiding something, when you can just do it purely with css/less breakpoints & display:none

Reply:
Especially when `wideScreen` state seems to not be used anywhere :)

Reply:
In this state this cannot be accepted to main (hard to maintain, hardcoded breakpoint logic creates inconsistency against SUI responsive classes)